peachjam_search/views.py

```diff
@@ -192,6 +192,14 @@ def build_basic_queries(self, request, view):
             for field in query_fields
         ]
 
+        if " " in self.query:
+            # do optimistic match-phrase queries for multi-word queries
+            for field, options in view.search_fields.items():
+                query = {"query": self.query, "slop": 0}
+                if "boost" in (options or {}):
+                    query["boost"] = options["boost"]
+                queries.append(MatchPhrase(**{field: query}))
```

Comment:
is this similar to adding quotes on searches? but we now enforce it on the backend

Reply:
yes, it's saying : try this search as a phrase and boost if it matches, but don't fail if it doesn't